### PR TITLE
New readme

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,118 +1,173 @@
...
 
-<img src="https://raw.githubusercontent.com/cdimascio/watson-nlc-spam/master/assets/watson-nlc.png"  width="200"  align="right" /></br>
+# Natural Language Classifier email spam classifier
 
-Expanding upon a previous sample app, built with Watson Natural Language Classifier by Carmine DiMascio, this new sample application will be focused on phishing emails and identifying them as spam. Note: Watson Natural Language Classifier can be trained to identify various types of emails, given that it is trained with the proper dataset. In this case we will be using the Enron emails as training data.
+In this Code Pattern, we will build an app that classifies email, either labeling it as "Phishing", "Spam", or "Ham" if it does not appear suspicious. We'll be using IBM Watson Natural Language Classifier (NLC) to train a model using email examples from an [EDRM Enron email dataset](https://www.edrm.net/resources/data-sets/edrm-enron-email-data-set/). The custom NLC model can be quickly and easily build in the Web UI, deployed into our nodejs app using the [Watson Developer Cloud Nodejs SDK](https://github.com/watson-developer-cloud/node-sdk), and then run from a browser.
```
Comment (rhagarty) [inline README.md]: "easily built"

File: README.md [hunk 2]
```diff
@@ -1,118 +1,173 @@
...
-  "https://gateway.watsonplatform.net/natural-language-classifier/api/v1/classifiers"
-    ```
-#### Second Method 
+![video-to-gif](doc/source/videos/dragNdropDataNLC.gif)
 
-Access the GUI based tooling by logging into your [IBM Cloud Dashboard](https://console.us-east.bluemix.net/dashboard/services). 
+* Click the `Train model` button to begin training. The model will take around an hour to train.
 
-In the Services area, click your Natural Language Classifier service tile to open the instance dashboard. Once in the service dashboard, click `Open tool` and upload your training data.
+* To check the status of the model, and access it after it trains, go to your project in the `Assets` tab, `Models` section. The model will show up when it is ready. Double click to see the `Overview` tab.
```
Comment (rhagarty) [inline README.md]: suggestion -  "`Assets` tab of the `Models` section."

File: server/manifest.yml [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
   domain: mybluemix.net
-  name: nlc-email-spam
-  host: nlc-email-spam
-
-  # disk_quota: 1024M
+  name: nlc-email-phishing
+  host: nlc-email-phishing
+  disk_quota: 1024M
+  random-route: true
   env:
```
Comment (rhagarty) [inline server/manifest.yml]: sure you want this set to true?

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md]: done

File: server/manifest.yml (hunk 3 above)
Comment (scottdangelo) [reply server/manifest.yml]: no. This isn't working yet anyway. I'll just remove it from the patch set

**Pull request merged**